index.html

```diff
@@ -53,6 +53,40 @@ <h2 class="connect">LET’S CONNECT</h2>
     </div>
 
   </section>
+  
+  <section class="work-section">
+    <div class="card_work">
+      <img class="snapshot" src="images/Snapshoot Portfolio.png" alt="">
+      <div class="left_block">
+        <h1 class="tonic-text">Tonic</h1>
+        <div class="frame-2">
+          <h4 class="canopy">CANOPY</h4>
+          <img class="count" src="images/Counter.png" alt="">
+          <h4 class="role">Back End Dev</h4>
+          <img class="count" src="images/Counter.png" alt="">
+          <h4 class="year">2015</h4>
+        </div>
+        <h3 class="daily_text">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</h3>
+        <ul class="tags">
+          <li>
+            <button class="html_but"><h3 class="html_text">html</h3></button>
+          </li>
+          <li>
+            <button class="css_but"><h3 class="css_text">css</h3></button>
+          </li>
+          <li><button class="java_but"><h3 class="java_text">javaScript</h3></button></li>
+
+        </ul>
+    
+        <div class="but container">
+          <button class="Action_but"><h1 class="text_project">See Project</h1></button>
+        </div>
+  
+      </div>
+
+    </div>
+    
+  </section>
```

Comment:
According to the design for this template, your work section should include four individual project cards aligned with display: grid. However, I can only see one project card. Kindly check and make these changes to conform to the figma design. :pray: